src/prompts/text.rs

```diff
@@ -65,6 +65,11 @@ pub struct Text<'a> {
     /// Message to be presented to the user.
     pub message: &'a str,
 
+    /// The text that will be inputed in the search bar at start.
+    /// If you want to show an example of what can be typed look at [placeholder].
+    /// If you want to set a default value when the user type nothing look at [default].
+    pub text: Option<&'a str>,
```

Comment:
```suggestion
    /// Initial value of the prompt's text input.
    ///
    /// If you want to set a default value for the prompt, returned when the user's submission is empty, see [default].
    pub initial_value: Option<&'a str>,
```

Reply:
wdyt of this variable name?

Reply:
Yes, I like it more. 
I think the word `value` in the name is less error-prone :grinning: